state/runtime/evm/instructions_test.go

```diff
@@ -58,7 +58,7 @@ func testArithmeticOperation(t *testing.T, f instruction, test OperandsArithmeti
 
 	f(s)
 
-	assert.EqualValues(t, test.expectedResult.Uint64(), s.pop().Uint64())
+	assert.EqualValues(t, test.expectedResult, s.pop())
```

Comment:
WDYT?
```suggestion
	assert.Equal(t, test.expectedResult, s.pop())
```

Reply:
I'm okay with this. I'll make changes through the entire test to ensure consistency.